docs/history/changelog.md

```diff
@@ -188,6 +190,8 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 * [#1315](https://github.com/open-contracting/standard/pull/1315) Add rules on setting `id` and `date` for compiled releases to the merging specification.
 * [#1344](https://github.com/open-contracting/standard/pull/1344) Add contract suspension worked example.
 * [#1375](https://github.com/open-contracting/standard/pull/1375) Update guidance for empty fields in the merging documentation.
+* [#1401](https://github.com/open-contracting/standard/pull/1401) Replace "total value" with "value".
+
```

Comment:
Once is enough. It can be assumed that any change to the schema might be reflected in the documentation.

```suggestion
```

Reply:
Maybe worth adding a point to https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/changelog_style_guide.html ? Is the rule "If you change the schema changelog, don't also change the documentation changelog?" 

We could also rename the "Schema" section of the changelog to "Schema and corresponding documentation" and "Documentation" to "Documentation (only)"?

Reply:
I've now added this point:

> If changes are made to files under the ``schema`` directory, it is assumed that corresponding changes were made to files under the ``docs`` directory. Do not add an entry under the "Documentation" heading if the changes directly correspond to the entry under the "Codelists" and/or "Schema" headings.